crates/sovereign-sdk/full-node/sov-stf-runner/src/verifier.rs

```diff
@@ -39,7 +39,6 @@ where
         zkvm: Zk,
         pre_state: Stf::PreState,
     ) -> Result<(), Da::Error> {
-        println!("Running sequencer commitments in DA slot");
```

Comment:
I believe these could be tracings

Reply:
no, tracing doesnt work inside zk context

and this prints are useful :) keep it pls